Argo/Operators/DecodedOperators.swift

```diff
@@ -68,3 +68,24 @@ public func <|><A>(lhs: Decoded<A>, rhs: Decoded<A>) -> Decoded<A> {
   }
   return rhs
 }
+
+// MARK: Failure coalescing operator
+
+/**
+  return the unwrapped value of the `Decoded` value on the left if it is `.Success`, otherwise return the default on the right
+
+  - If the left hand side is successful, this will return the unwrapped value from the left hand side argument
+  - If the left hand side is unsuccesful, this will return the default value on the right hand side
+
+  - parameter lhs: A value of type `Decoded<A>`
+  - parameter rhs: A value of type `A`
+
+  - returns: A value of type `A`
+*/
+
+public func ?? <A>(lhs: Decoded<A>, rhs: A) -> A {
```

Comment:
In Swift 2, these are the definitions for the Optional `??`:

```swift
@warn_unused_result
@rethrows public func ??<T>(optional: T?, @autoclosure defaultValue: () throws -> T?) rethrows -> T?

@warn_unused_result
@rethrows public func ??<T>(optional: T?, @autoclosure defaultValue: () throws -> T) rethrows -> T
```
The first one being the same as what we consider Alternative, `<|>`. Since `Decoded` is our form of error handling, I don't think we need any of the `throws` or `rethrows` stuff. We should probably add `@autoclosure` to alternative as well. AND AND `@noescape` to all the other monadic operators for `Decoded`. Another PR though.